wpilibjIntegrationTests/src/main/java/edu/wpi/first/wpilibj/DIOCrossConnectTest.java

```diff
@@ -143,7 +144,7 @@ public void testDIOpulseWidthModulation() {
    * edu.wpi.first.wpilibj.AbstractInterruptTest#giveInterruptableSensorBase()
    */
   @Override
-  InterruptableSensorBase giveInterruptableSensorBase() {
+  DigitalSource giveInterruptableSensorBase() {
```

Comment:
```suggestion
  DigitalSource giveInterruptibleSensorBase() {
```

Reply:
Fixing this by renaming the function completely.